cinnabar/plotting.py

```diff
@@ -83,11 +92,11 @@ def _master_plot(
     fig = plt.figure(figsize=(figsize, figsize))
     plt.subplots_adjust(left=0.2, right=0.8, bottom=0.2, top=0.8)
 
-    plt.xlabel(f"{xlabel} {quantity} / " + units)
-    plt.ylabel(f"{ylabel} {quantity} / " + units)
+    plt.xlabel(f"{xlabel} {quantity} ({units})")
+    plt.ylabel(f"{ylabel} {quantity} ({units})")
 
-    ax_min = min(min(x), min(y)) - 0.5
-    ax_max = max(max(x), max(y)) + 0.5
+    ax_min = min(min(x), min(y)) - 1.0
+    ax_max = max(max(x), max(y)) + 1.0
```

Comment:
What is this change for?

Reply:
In the PR description, its: 
Make the minimum and maximum axes of the plot larger

Reply:
In my plots, this change is necessary because otherwise some of the error bars are cut off, so I extended the plot to make it bigger

Reply:
The more general fix for this is probably to adjust `ax_min` and `ax_max` to find the min and max of the error bars, not free energies.

Reply:
This is the only change (as far as I can tell) that changes behavior in a way where a plot made with this PR would look different than a plot made with the current code. How about you add another argument called `error_padding` or something that defaults to `0.5` but then a user can pass in whatever value they want.

Reply:
This is done!